great_expectations/experimental/datasources/sqlite_datasource.py

```diff
@@ -108,7 +109,7 @@ class SqliteDatasource(SQLDatasource):
     # right side of the operator determines the type name
     # left side enforces the names on instance creation
     type: Literal["sqlite"] = "sqlite"  # type: ignore[assignment]
-    connection_string: SqliteConnectionString
+    connection_string: str
```

Comment:
```suggestion
    connection_string: SqliteDsn
```
??

BTW this is very slick. Nice work 🚀 
The errors will be beautiful

Reply:
I had this like this and planned to add it back once I got around to fixing the typing errors.